### ISSUE TITLE
cache not cleaned for global write buffer (extend vpool)

### ISSUE THREAD

**Issue by jeroenmaelbrancke** [openvstorage/framework#1198]
Problem:
The cached value from global write buffer is not been cleaned when i change the write role.

Version:
```
openvstorage                         2.7.6-rev.4333.10879df-1        amd64        openvStorage
openvstorage-backend                 1.7.6-rev.817.91abb7c-1         amd64        openvStorage Backend plugin
openvstorage-backend-core            1.7.6-rev.817.91abb7c-1         amd64        openvStorage Backend plugin core
openvstorage-backend-webapps         1.7.6-rev.817.91abb7c-1         amd64        openvStorage Backend plugin Web Applications
openvstorage-core                    2.7.6-rev.4333.10879df-1        amd64        openvStorage core
openvstorage-hc                      1.7.6-rev.817.91abb7c-1         amd64        openvStorage Backend plugin HyperConverged
openvstorage-sdm                     1.6.6-rev.449.77bd347-1         amd64        Open vStorage Backend ASD Manager
openvstorage-webapps                 2.7.6-rev.4333.10879df-1        amd64        openvStorage Web Applications
```

![extend_vpool](https://cloud.githubusercontent.com/assets/2900612/20556706/9147c6d6-b168-11e6-9b28-74f6fce7b6e3.png)

**Comment by kvanhijf** [comment]
Issue was caused by first assigning 2 devices with a WRITE role --> In total there was 1043GiB of write buffer space
The wizard was cancelled and 1 WRITE role was removed --> Only 111GiB was left for write buffer space, but the wizard was still showing the previous value (See screenshot)

**Comment by kvanhijf** [comment]
Fixed by https://github.com/openvstorage/framework/issues/1308
While implementing the multiple proxies, an optimization for the add vPool wizard was done which also included this fix.

Included in openvstorage-2.7.9.1

**Comment by kvanhijf** [comment]
@openvstorage/qa : openvstorage-2.7.10.1
